README.md

```diff
@@ -2,8 +2,6 @@
 Let's collect the data of Woman on GitHub, who inspired you to code more, learn more.
 
 **Note:** Please maintain the below list in alphabetical order.
-
-[alicetragedy](https://github.com/alicetragedy)
```

Comment:
Rectified the error
![err1](https://cloud.githubusercontent.com/assets/7047079/12530267/43a0fd84-c1ff-11e5-9dfc-45c97f425081.png)
![err2](https://cloud.githubusercontent.com/assets/7047079/12530268/43aa33cc-c1ff-11e5-9350-fe4414eaf8e0.png)